build/Dockerfile

```diff
@@ -380,7 +380,6 @@ RUN --mount=type=secret,id=nginx-repo.crt,dst=/etc/ssl/nginx/nginx-repo.crt,mode
 	agent.sh \
 	; fi
 
-
```

Comment:
@jjngx this change is forcing the whole CI run, think we can omit it and leave it as a docs only PR